ui/app/components/job-page/parts/recent-allocations.js

```diff
@@ -20,7 +23,7 @@ export default Component.extend({
 
   actions: {
     gotoAllocation(allocation) {
-      this.transitionToRoute('allocations.allocation', allocation);
+      this.router.transitionTo('allocations.allocation', allocation.id);
```

Comment:
any specific reason to reload the record?

Reply:
I found that without the `.id`, I got this error:
> You didn't provide enough string/numeric parameters to satisfy all of the dynamic segments for route

My belief is that it’s because the router is expecting `allocation_id`, but the model only has an `id`. Changing the router seemed like Too Much for this small fix.